.vscode/settings.json

```diff
@@ -1,3 +1,16 @@
 {
-    "liveServer.settings.port": 5501
+  "liveServer.settings.port": 5501,
+  "C_Cpp_Runner.warnings": [
+    "-Wall",
+    "-Wextra",
+    "-Wpedantic"
+  ],
+  "C_Cpp_Runner.compilerArgs": [],
+  "C_Cpp_Runner.includePaths": [],
```

Comment:
unnecessary changes 🤕

Reply:
yaah amended it!

Reply:
you didn't removed it why ?

Reply:
I think I took it wrong then.. do I have to completely delete this file? or just delete the code which you are specifying above?

Reply:
ok I did something .. can you pls check this time I removed it properly ?

Reply:
No it's not correct only this line should be there 
```
{
 "liveServer.settings.port": 5501
}
```

rest lines aren't related to this project

Reply:
yaah i did it only I guess..
![Screenshot (252)](https://user-images.githubusercontent.com/85383286/125774114-660225cc-3c94-4df6-86d8-fea5b49e5c0f.png)

Reply:
ya it's done now.